locallib.php

```diff
@@ -289,14 +295,34 @@ function create_report_transcription($transcription) {
 /**
  * Creates a button with identical id and class
  *
- * @param string $id
+ * @param string $id id and class of the button
  * @param string $text of the button
  */
 function create_button($id, $text) {
     $out = html_writer::tag('button', $text, array('id' => $id, 'class' => $id));
     return $out;
 }
 
+/**
+ * Creates navigation buttons with identical id and class
+ *
+ * @param string $phase name of the phase of the assignment
+ */
+function create_nav_buttons($phase) {
+    $out = html_writer::start_div('navbuttons');
+    if ($phase == 'info') {
+        $out .= create_button('nextButton', 'Next >');
```

Comment:
Could all of these buttons be as link elements? Ref: http://componentlibrary.moodle.com/admin/tool/componentlibrary/docspage.php/moodle/components/buttons/ and under `Button links`. This would disable need for Javascript as link generator. As a style stand point, this would follow coloring theme set up in the Moodle's settings and match with Moodle's other buttons.
`<a href="link" class="btn btn-primary">Button</a>`